<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmButtonsDisplay</class>
 <widget class="QWidget" name="frmButtonsDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmButtonsDisplay</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1">
   <item>
    <widget class="QLabel" name="mapLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonsFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>复位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeButton_Circle">
        <property name="text">
         <string>圆形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeButton_Police">
        <property name="text">
         <string>警察</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeButton_Bubble">
        <property name="text">
         <string>气泡</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeButton_Bubble2">
        <property name="text">
         <string>气泡2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeButton_Info">
        <property name="text">
         <string>消息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeButton_Info2">
        <property name="text">
         <string>消息2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_HorizonalLine">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Green">
        <property name="text">
         <string>绿色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Blue">
        <property name="text">
         <string>蓝色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Red">
        <property name="text">
         <string>红色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Grey">
        <property name="text">
         <string>灰色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Black">
        <property name="text">
         <string>黑色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Purple">
        <property name="text">
         <string>紫色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton_Yellow">
        <property name="text">
         <string>黄色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Movable">
        <property name="text">
         <string>可移动</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
